uaa-server/README.adoc

```diff
@@ -0,0 +1,13 @@
+= steeltoeoss/sample-uaa-server
+:toc: preamble
+:toclevels: 1
+:!toc-title:
+:linkattrs:
+
+SteeltoeOSS https://github.com/cloudfoundry/uaa[CloudFoundry User Account and Authentication (UAA)] docker image.
+
+== Running
+
+----
+$ docker run -d -it -p 8080:8080 steeltoeoss/sample-uaa-server
```

Comment:
Just checking you intended example to run detached

Reply:
I was running detached locally, but I also want to be consistent with other docs in this repo, so I'll take that out in my upcoming update.